internal/ingress/controller/template/template.go

```diff
@@ -314,7 +315,7 @@ func atLeastOneNeedsRewrite(input interface{}) bool {
 
 // buildLocation produces the location string, if the ingress has redirects
 // (specified through the nginx.ingress.kubernetes.io/rewrite-target annotation)
-func buildLocation(input interface{}, rewrite bool) string {
+func buildLocation(input interface{}, useRegex bool) string {
```

Comment:
similar to above comment, `useRegex` -> `enforceRegexModifier`, or simply `enforceRegex`